sources/packages/api/src/esdc-integration/e-cert-full-time-integration/e-cert-files/e-cert-response-record-identification.ts

```diff
@@ -0,0 +1,23 @@
+import { RecordTypeCodes } from "../models/e-cert-full-time-integration.model";
+
+export class ECertResponseRecordIdentification {
+  constructor(line: string, lineNumber: number) {
+    this.transactionCode = line.substr(0, 3) as RecordTypeCodes;
+    this.line = line;
+    this.lineNumber = lineNumber + 1;
+  }
+
+  /**
+   * transactionCode of the line.
+   */
+  public readonly transactionCode: RecordTypeCodes;
```

Comment:
I would say that this variable would be better called _recordTypeCode_.
I believe that the term transaction code came from CRA integration because of the documentation referer to it as a transaction.
For MSFAA and E-Cert, the document refers to it as the record type, so I would recommend keeping it aligned to what the documentation uses.